src/routes/(authed)/profile/+page.svelte

```diff
@@ -1,9 +1,19 @@
+<script lang="ts">
+	import { goto } from '$app/navigation';
+
+	export let data;
+
+	let firstname = data.user.firstname;
+	let lastname = data.user.lastname;
+	let email = data.user.email;
+</script>
+
 <div id="page" class="h-screen bg-background">
 	<div
 		id="header"
 		class="relative flex h-16 w-screen flex-row items-center justify-between rounded-b-xl border-2 backdrop-blur-sm dark:border-gray-600 dark:bg-gray-900 dark:text-gray-300"
 	>
-		<button class="relative ml-1 flex rounded-full bg-background p-1">
+		<button class="relative ml-1 flex rounded-full bg-background p-1" on:click={() => goto("/")}>
```

Comment:
is this supposed to goto home screen or profile

Reply:
Home screen